addon/coniks

```diff
@@ -0,0 +1 @@
+../chrome/content/coniks/
```

Comment:
Yeah, I'm not sure why this doesn't work.

I think originally I just tried putting a link to `content/` in `addon/` and eventually gave up and linked the files individually.